backend/views/user_view.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import viewsets
+from serializers.user_serializer import UserSerializer
+from custom_models.models import User
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    # GETS USER LIST
+    queryset = User.objects.all()  # same as 'SELECT * FROM backend_myuser;'
+    serializer_class = UserSerializer
+    # allows only GET requests
+    http_method_names = [
+        "get"
+    ]  # TODO: (fnormand01) disable GET request as soon as we deploy

```

Comment:
I agree! I removed the whole user view, since we have swagger now.
See commit #https://github.com/code-for-canada/project-thundercat/pull/275/commits/3aea56f1dcb9b41e78993255e0f9dd4706213dea